components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/update-claim.jsp

```diff
@@ -181,11 +182,11 @@
                             ClaimMappingDTO[] claims = claimMappping[i].getClaimMappings();
                 %>
                 <a href="#" class="icon-link" style="background-image:url(../claim-mgt/images/delete.gif);"
-                   onclick="removeItem('<%=dialectUri%>','<%=claimUri%>','<%=claims.length%>'  );return false;"><fmt:message
+                   onclick="removeItem('<%=Encode.forJavaScriptAttribute(dialectUri)%>','<%=Encode.forJavaScriptAttribute(claimUri)%>','<%=Encode.forJavaScriptAttribute(String.valueOf(claims.length))%>'  );return false;"><fmt:message

```

Comment:
Do not encode the integer claims.length which is already type safe. This might break the javascript function in case it's expecting an int